### DISCUSSION TITLE
Difference between dagit reload buttons

### DISCUSSION THREAD

**Discussion by bendnorman** [dagster-io/dagster#13819] [Q&A]
1. What is the difference between the "reload location" and the "Reload Definitions" buttons in Dagit? 

"Reload location":
![image](https://user-images.githubusercontent.com/17532695/233487261-89047f86-6f94-4db2-a722-46134067461d.png)

"Reload Definitions":
![image](https://user-images.githubusercontent.com/17532695/233487318-9a520b3f-54c6-4218-96d6-0178d5be6b90.png)

2. Why does the "Reload Definitions" button appear for asset groups but not jobs?
3. Does the code location only need to be reloaded when a NEW asset is added (or removed?) Or also when an existing asset definition is changed?

**Comment by sryza** [comment]
1. I think they should do the same thing
2. @braunjj - thoughts on this?
3. If you're just changing what's inside the decorated function, you don't need to reload, but if you're changing the name or the dependencies or any of the arguments to the `@asset` decorator, you need to reload.
